src/templates.rs

```diff
@@ -181,3 +185,13 @@ impl<'v> FromFormValue<'v> for FormError {
         Ok(FormError::from(form_value))
     }
 }
+
+pub const RESERVED_WORDS: &[&str] = &["new", "start", "rcos", "edit"];
+
+pub fn is_reserved(word: &str) -> Result<&str, FormError> {
```

Comment:
In this function, add a check to see if the word is just a number? Having a number could cause unintended effects.

Reply:
Did this in https://github.com/rcos/observatory-new/pull/98/commits/232a1de540eafd3e49d2fe97a679aa7f453fcbab it was only a one-line change.